architecture/core_concepts/deployments.adoc

```diff
@@ -69,6 +69,8 @@ spec:
 2. The label selector of the pod to run.
 3. A template for the pod the controller creates.
 4. Labels on the pod should include those from the label selector.
+5. The maximum name length is 63 characters.
```

Comment:
It might be worth adding the phrase "after expanding any parameters" after "maximum name length".  This specific example doesn't use parameters, but it does use a template, and the fact that these limits apply *after* parameter expansion should be documented in the context of templates.  A better location to call this fact out might be [the documentation on templates in the Developer Guide](https://github.com/openshift/openshift-docs/blob/master/dev_guide/templates.adoc#templates-parameters)—except the examples there don't use parameters in the name field.  Hm...